pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "honeycomb-opentelemetry"
-version = "0.1.1a3"
+version = "0.1.0b0"
```

Comment:
🤔   I was expecting maybe `0.1.2b0` or `0.2.0b0`. Why go downwards with the patch version?